src/main/java/website/frontrow/game/Game.java

```diff
@@ -214,6 +233,36 @@ public void levelWon()
     @Override
     public void levelLost()
     {
+        gameOver();
         stop();
     }
+    
+    /**
+     * Returns whether or not all players are dead.
+     * @return b boolean
+     */
+    public boolean playersDead()
+    {
+    	if(!this.isRunning())
+    	{
+    		return false;
+    	}
+    	boolean b = true;
+    	for(Player p : this.players)
+    	{
+    		if (p.playerAlive())
+    		{
+    			b = false;
```

Comment:
Wouldn't it be easier to just use "return false;" and at the bottom use "return true" this will save you having to declare a variable for it.